pkg/test/framework/components/istio/operator.go

```diff
@@ -471,7 +492,7 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster resource.Clust
 		installSettings = append(installSettings, "--set", fmt.Sprintf("values.%s=%s", k, v))
 	}
 
-	if c.environment.IsMulticluster() {
+	if c.environment.IsMulticluster() && !c.isExternalControlPlane() {
```

Comment:
why this change? Should the cluster name always be set?

Reply:
in `external control plane` model, we do not need to create the remote secret as well as all this special logic in this snippet. As it is using the KUBECONFIG from the `config` cluster directly. So need skip all this logic.

Reply:
It doesn't hurt tho, right? I'm just saying that it might be better to remove as many conditional branches as possible, for the sake of readability

Reply:
It will hurt actually, for example it will create `values.global.remotePilotAddress` , this does not work for `external control plane` mode